game/templates/game/board.html

```
@@ -339,7 +351,11 @@
             <p id="confirmMessage" style="color: #aaa; margin-bottom: 20px; font-size: 0.9rem; letter-spacing: 0.5px;">
                 Your current progress will be lost.<br>Are you sure you want to start a new game?
             </p>
+
+            <div id="confirmTimerContainer" style="margin-bottom: 20px; text-align: left;">
+
             <div id="confirmTimerContainer" style="display: none; margin-bottom: 20px; text-align: left;">
```

HTMLHint (1.9.2)

[error] The id value [ confirmTimerContainer ] must be unique.

(id-unique)